Dockerfile

```diff
@@ -39,13 +39,17 @@ RUN echo 'deb http://deb.debian.org/debian bullseye-backports main' \
       ninja-build \
       pkg-config \
       robin-map-dev \
+      curl \
       wget && \
     wget "https://apache.jfrog.io/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb" && \
     apt-get -y --no-install-recommends install \
       ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb && \
     apt-get update && \
     apt-get -y --no-install-recommends install libarrow-dev=9.0.0-1 libprotobuf-dev libparquet-dev=9.0.0-1 && \
-    rm -rf /var/lib/apt/lists/* *.deb
+    rm -rf /var/lib/apt/lists/* *.deb && \
+    curl -fsSL https://deb.nodesource.com/setup_18.x | bash - && \
+    apt-get install -y nodejs && \
+    corepack enable
```

Comment:
This has to come before the `rm -rf /var/lib/apt/lists/*` in order to avoid inflating the generated image size.